frontend/src/locale/index.ts

```diff
@@ -120,6 +120,60 @@ Let's get in touch to find the best solution for your business' needs!`,
 
     "profile.welcome": "Ciao {name}!",
     "profile.myProfile": "My Profile",
+
```

Comment:
@alessiamarcolini here
For every status, in `my-grant` page we will display `<status>.nextSteps`.